### PR TITLE
Removed -SNAPSHOT from 1.6

### PR DESCRIPTION
**Pull request by Nishith**

### CODE REVIEW

File: database-plugins/pom.xml [hunk 1]
```diff
@@ -20,12 +20,12 @@
   <parent>
     <artifactId>hydrator-plugins</artifactId>
     <groupId>co.cask.hydrator</groupId>
-    <version>1.6.0-SNAPSHOT</version>
+    <version>1.6.0</version>
   </parent>
 
   <properties>
     <!-- properties for script build step that creates the config files for the artifacts -->
```
Comment (anwar6953) [inline database-plugins/pom.xml:27]: any idea why some pom files have this `app.parents`, but like cassandra-plugins/pom.xml doesn't?

Comment (Nishith) [reply database-plugins/pom.xml:27]: Don't know

Comment (sreevatsanraman) [reply database-plugins/pom.xml:27]: Let's open a JIRA to cleanup for next release.

**Comment by sreevatsanraman** [comment]
LGTM

File: database-plugins/pom.xml (hunk 1 above)
Comment (anwar6953) [reply database-plugins/pom.xml:27]: https://issues.cask.co/browse/HYDRATOR-1400

**Pull request merged**